src/main/java/fr/romitou/mongosk/skript/expressions/ExprCollection.java

```diff
@@ -41,9 +41,13 @@ public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelaye
     protected MongoCollection[] get(Event e) {
         String name = exprName.getSingle(e);
         MongoDatabase database = exprDatabase.getSingle(e);
-        return (name == null || database == null)
-                ? new MongoCollection[0]
-                : new MongoCollection[]{database.getCollection(name)};
+        try {
+            return (name == null || database == null)
+                    ? new MongoCollection[0]
+                    : new MongoCollection[]{database.getCollection(name)};
+        } catch (IllegalArgumentException ex) {
+            return null; // A MongoDB collection with that name doesn't exist!
```

Comment:
Don't return null when you can
```suggestion
            return new MongoCollection[0]; // A MongoDB collection with that name doesn't exist!
```